front/client/components/Article.vue

```diff
@@ -49,21 +51,49 @@
           </div>
           <div class="article-section notes-box">
             <p class="article-section-title">Notes</p>
-            <div v-html="value.notes.html"></div>
+            <div class="notes-settings">
+              <a href="" class="notes-settings-btn">expand notes</a>
+              <a href="" class="notes-settings-btn" style="display: none;">collapse notes</a>
+              <a href="" class="notes-settings-btn">edit notes</a>
+            </div>
+            <div v-for="(value, index) in value.toc" :key="index">
+              <ul v-for="(value, index) in value" :key="index">
+                <li class="notes-toc-item" v-if="value.content !== undefined">
+                  <a :href="`#${value.slug}`" @click="expandNotes()">
```

Comment:
@click="expandNotes" you dont have to call function manually, vue does it for you